src/user-research/index.md

```diff
@@ -17,7 +17,7 @@ Our principles are not rules. They guide our work, keep us improving as a team,
 
 1. **Help teams understand and value people**
 
-   Researchers at dxw help our teams to build a deep understanding of the people who use our services, and to value and act on that understanding. That’s how we contribute to good decisions and public services that work well for the people who need them.
+   Researchers at dxw help our teams to build a deep understanding of the people who use our services, and to value and act on that understanding. So our teams can make good decisions and create public services that work well for the people who need them.
```

Comment:
This one seems a little more jarring than the `And`-starting sentences to me. Perhaps because it's more common to start with `And` than `So`

Reply:
Let's just go back to the sentence we had before. I only changed it to avoid saying 'help our teams' twice in a paragraph.

"We know that’s the best way to help teams make good decisions and create public services that work well for the people who need them."

Reply:
The substantive change was to add ", and to value and act on that understanding." And everyone's happy with that.